web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -55,39 +51,33 @@ In the Overview tab:
 * Secret
 * Status
 * Storage class
-* Capacity data: Usage/ OBC quota/ pie chart for available
-* Bucket region
 * Object Bucket that was created by this claim
-[OBC quotas are out of scope for OCS 4.2: remove quota label for 4.2]
 
 Object bucket claim data (for application consumption):
 * Bucket name 
 * Access Key
 * Secret Key
 * Endpoint
+
 *all data is masked and can be revealed by clicking on “Reveal Values”
 
 ![OBC details](img/OBC-overview.png)
```

Comment:
Should the design be updated to reflect dropping capacity and region?

Reply:
@lizsurette I'm pretty sure it's updated, do you see anything in the design that is with the capacity and region?

Reply:
@shirimordechay Hmm for some reason I'm still seeing the older designs and not newer ones. Maybe a naming issue? Could just be my issue :)

@matthewcarleton - Can you do me a favor and see if you see the "Used" field in the latest mockup here?

In any case, I think this is in a good spot to merge and we can put up a new quick PR to update if the designs are older.

Reply:
@lizsurette yup I see it. I've had weird image caching issues before on GH, might be what's happening.

Reply:
Maybe it's that caching issue! I'll merge and see if the images are the right ones. If not, we can do an update PR from there.